mne/viz/raw.py

```diff
@@ -59,7 +59,7 @@ def _update_raw_data(params):
         data[di] /= params['scalings'][params['types'][di]]
         # stim channels should be hard limited
         if params['types'][di] == 'stim':
-            data[di] = np.minimum(data[di], 1.0)
+            data[di] /= float(max(data[di]))
```

Comment:
The trigger values were basically always plotted at amplitude of 1 unless there are overlaps. This fixes it. This will also make small trigger values harder to see (imagine trigger values of 1 and 512 for instance). See if you agree with the change.